benchmarks/benchmarks.py

```diff
@@ -83,3 +86,248 @@ def _circular_standard_deviation(samples, high=2 * np.pi, low=0, axis=None):
         except ImportError:
             data = np.random.randn(10000, 1000)
             return circstd(data)
+
+
+class Fast_Kde_1d:
```

Comment:
Is there a way to do the benchmarks calling arviz functions? Otherwise, if somebody were to change the source of `_fast_kde` this file would also have to be modified for benchmarks to be up to date.

Reply:
Build fails :(

Reply:
What does build fails means?